ppr-ui/src/components/mhrRegistration/YourHome/OtherInformation.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <v-card flat class="white pb-6 pt-6 pr-10 pl-8 rounded">
+      <v-row>
+        <v-col cols="2">
+          <label class="generic-label ml" for="">Other</label>
+        </v-col>
+        <v-col cols="10">
+          <v-textarea
+            v-model="otherRemarks"
+            filled
+            :error-messages="errorMessages.otherRemarks"
+            name="name"
+            counter="140"
+            placeholder="Enter other details about the home (Optional)"
```

Comment:
Designs look like they call for a `hint` with a `persistent-hint` property.
I could be wrong but i believe placeholder is gone once you start typing in the input?

Reply:
To me, the design looks like a placeholder. It will be gone after they type.
<img width="938" alt="Screen Shot 2022-06-22 at 8 26 40 AM" src="https://user-images.githubusercontent.com/2333290/175069906-a2f2af89-6f07-44f1-88d4-0b8179d2c6dc.png">

Reply:
Tracey confirmed the text should stay. Will update it.